tests/test_classification.py

```diff
@@ -178,15 +182,40 @@ def test_sparse_predict():
                                               'RandomForestClassifier',
                                               'AdaBoostClassifier',
                                               'MultinomialNB',
-                                              'KNeighborsClassifier'],
+                                              'KNeighborsClassifier',
+                                              'RidgeClassifier',
+                                              'MLPClassifier'],
                                              [(0.45, 0.52), (0.52, 0.5),
                                               (0.48, 0.5), (0.49, 0.5),
-                                              (0.43, 0), (0.53, 0.57)]):
+                                              (0.43, 0), (0.53, 0.57),
+                                              (0.49, 0.49), (0.48, 0.5)]):
         yield check_sparse_predict, learner_name, expected_scores[0], False
         if learner_name != 'MultinomialNB':
             yield check_sparse_predict, learner_name, expected_scores[1], True
 
 
+def test_mlp_classification():
+    train_fs, test_fs = make_classification_data(num_examples=600,
+                                                 train_test_ratio=0.8,
+                                                 num_labels=3,
+                                                 num_features=5)
+
+    # train an AdaBoostRegressor on the training data and evalute on the
+    # testing data
+    learner = Learner('MLPClassifier', model_kwargs={'solver': 'lbfgs'})
```

Comment:
Should this comment say "train an MLPClassifier on the training data and evaluate on the testing data" ?